.github/workflows/publish-cdn.yml

```diff
@@ -0,0 +1,27 @@
+name: Publish to CDN
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version, without v prefix'
+        required: true
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      packages: write
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.inputs.version }}
+      - name: Use Node.js 14.x
+        env:
+          AWS_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY }}
```

Comment:
I think both the `AWS_` secret keys could have a more qualified name, perhaps expand the prefix to `CDN_PUBLISH_AWS_`?

(as our use of secrets expands for this repository, over time, then we may have other AWS secrets come in that could clash... and 'renaming secrets' isn't a thing in GitHub so may as well not set ourselves up for that pain)

Reply:
Fixed in https://github.com/ably/ably-js/pull/889/commits/04ff346c729c68e371eeada6a4f41aeaa13c8882